apps/vite/src/pages/About.tsx

```
@@ -1,240 +1,240 @@
+          >
+            Nepali Calendar.
+          </a>
+        </p>
+      </div>
+      <div className="mb-8 dark:text-white">
+        <b> Github Contributors:</b>
+        <a href="https://github.com/PoskOfficial/Miti/graphs/contributors">
+          <img
+            className="mt-3"
+            src="https://contrib.rocks/image?repo=PoskOfficial/Miti"
+          />
```

Biome

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)